ci-operator/jobs/shiftstack/bugwatcher/shiftstack-bugwatcher-main-presubmits.yaml

```diff
@@ -0,0 +1,98 @@
+presubmits:
+  shiftstack/bugwatcher:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - main
+    cluster: build01
+    context: ci/prow/images
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-shiftstack-bugwatcher-main-images
+    rerun_command: /test images
+    spec:
+      containers:
+      - args:
+        - --gcs-upload-secret=/secrets/gcs/service-account.json
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --report-credentials-file=/etc/report/credentials
+        - --target=[images]
+        command:
+        - ci-operator
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /secrets/gcs
+          name: gcs-credentials
+          readOnly: true
+        - mountPath: /etc/pull-secret
+          name: pull-secret
+          readOnly: true
+        - mountPath: /etc/report
+          name: result-aggregator
+          readOnly: true
+      serviceAccountName: ci-operator
+      volumes:
+      - name: pull-secret
+        secret:
+          secretName: registry-pull-credentials
+      - name: result-aggregator
+        secret:
+          secretName: result-aggregator
+    trigger: (?m)^/test( | .* )images,?($|\s.*)
```

Comment:
This regex is weird. Matches:

/test images
/test foo bar images
/test foo bar images,
/test foo bar images, foo blah whatever
/test foo images bar

I'm guessing its purpose is to match a test specified as a member of a space or comma separated list 🤷‍♂️

Reply:
the job is generated by ci-operator, I am not going to argue

Reply:
You can do this in a PR:

```plaintext
/test e2e-openstack e2e-openstack-upi e2e-openstack-ipi
```